Sources/XCHammer/XcodeTarget.swift

```diff
@@ -58,6 +58,15 @@ let DirectoriesAsFileSuffixes: [String] = [
     "lproj",
 ]
 
+func isBundleLibrary(_ ruleType: String) -> Bool {
+    // TODO: Remove deprecated rule
+    return ruleType == "objc_bundle_library" || ruleType == "apple_resource_bundle"
+}
+
+func isBundle(_ ruleType: String) -> Bool {
```

Comment:
Both the names of these methods follow to the objc names `objc_bundle`/ `objc_bundle_library`. Maybe since they added `_import` to `apple_bundle`, we call it `isBundleImport`